modules/xmpp/xmpp.js

```diff
@@ -1,7 +1,10 @@
-/* global $, Strophe */
+/* global $ */
 
 import { getLogger } from 'jitsi-meet-logger';
-const logger = getLogger(__filename);
+import { Strophe } from 'strophe.js';
+
+// Strophe plugins:
```

Comment:
pointless comment?

Reply:
I wanted to separate the plugins because we are just importing them without using exported value. If you think this is not useful I don't mind removing it.

Reply:
OK. I agreed! We have also our plugins and the comment is confusing. I'll remove it!